deploy/resources/v0.11.1/consolecli/console_cli_download_tkn.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: tkn
+spec:
+  description: The OpenShift Pipeline client `tkn` is a CLI tool that allows you to manage OpenShift Pipeline resources.
+  displayName: tkn - OpenShift Pipeline Command Line Interface (CLI)
+  links:
+    - href: https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-linux-amd64-latest.tar.gz
+      text: Download tkn for Linux
+    - href: https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-macos-amd64-latest.tar.gz
+      text: Download tkn for macOS
+    - href: https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-windows-amd64-latest.zip
+      text: Download tkn for Windows
```

Comment:
used latest because don't know the exact p12n version for the pipeline and in this [location](https://mirror.openshift.com/pub/openshift-v4/clients/tkn/) as of now nothing is there so just used latest
@vdemeester Let me know which version i can provide
@navidshaikh Thank you